test/helpers/prepare.js

```diff
@@ -0,0 +1,88 @@
+// Native
+const { join } = require('path')
+
+// Packages
+const strip = require('strip-indent')
+const { imageSync: getImageFile } = require('qr-image')
```

Comment:
What's the purpose of this?

Reply:
To generate the sample images for static deployments.